sqldelight-core-integration-tests/src/test/kotlin/com/example/QueryWrapper.kt

```diff
@@ -14,6 +14,8 @@ class QueryWrapper(
 
     val playerQueries: PlayerQueries = PlayerQueries(this, database)
     companion object : SqlDatabase.Helper {
+        override val version: Int = 1
```

Comment:
suuuuper nit: I think this generates a backing field _and_ a getter which dereferences the field. Can we switch it to `get() = 1` so that it's just a getter which returns the constant?